var/spack/repos/builtin/packages/ophidia-primitives/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class OphidiaPrimitives(AutotoolsPackage):
+    """Array-based primitives for the Ophidia framework"""
+
+    homepage = "https://github.com/OphidiaBigData/ophidia-primitives"
+    url = "https://github.com/OphidiaBigData/ophidia-primitives/archive/refs/tags/v1.7.1.tar.gz"
+    maintainers = ["eldoo", "SoniaScard"]
```

Comment:
@eldoo Are you okay with being added as a maintainer of this package?

Reply:
Yes, that is ok